### DISCUSSION TITLE
No winutil shortcut after format

### DISCUSSION THREAD

**Discussion by nhmf** [ChrisTitusTech/winutil#3069] [Q&A]
Hello I've used microwin to format but there is no shortcut on my desktop and I can't create on my own

**Comment by hashaski** [comment]
I'm having the same issue. Have you figured it out on how to make a shortcut?

**Reply by NotYourAverageGamer** [reply]
https://github.com/ChrisTitusTech/winutil/discussions/3127 or my comment below :)

**Comment by ManfaceDonkeh** [comment]
I am also having the same issue. after formatting. Windows 11 Pro, fully updated, not an insider. Winutil runs fine but the create a shortcut option has vanished. 

![WU0112](https://github.com/user-attachments/assets/7b6d7e25-37ec-4189-b982-dc4a35429c3b)

**Reply by NotYourAverageGamer** [reply]
https://github.com/ChrisTitusTech/winutil/discussions/3127 or my comment below :)

**Comment by NotYourAverageGamer** [comment]
The option to add a shortcut was removed due to anti-virus programs flagging and removing it. E.g. https://github.com/ChrisTitusTech/winutil/issues/3074, https://github.com/ChrisTitusTech/winutil/issues/2965

I previously posted a way to manually add a shortcut yourself if you choose to do so:
https://github.com/ChrisTitusTech/winutil/issues/2487#issuecomment-2265224487

You can manually create the shortcut by doing the following: (this is for win10 but win11 should be the same)

  - Right-click on your desktop and go to `New -> Shortcut`

  - It should ask for the location of the item, copy and paste the following:


```powershell
"C:\Program Files\PowerShell\7\pwsh.exe" -ExecutionPolicy Bypass -Command "Start-Process pwsh.exe -verb runas -ArgumentList '-Command "irm https://christitus.com/win | iex"'
```

  - Press Next/Enter

  - It should ask what you want to name the shortcut, enter `WinUtil`

  - Right-Click the new `WinUtil` shortcut on your desktop, and select `Properties`

  - In the `Shortcut` tab, select `Advanced...`

  - Tick the box for `Run as administrator`

  - Press OK then press OK again.

You should now be able to run WinUtil from that shortcut.

To fix the missing icon:

  - Right-Click the shortcut -> Properties -> Change Icon...

  - Below "Look for icons in this file:", replace everything there with:

```
%USERPROFILE%\AppData\Local\winutil\cttlogo.ico
```

Press Enter twice, then click `OK`

You should now have a manually created version of the shortcut that WinUtil would usually make :)

Here is the shortcut icon:
![cttlogo](https://private-user-images.githubusercontent.com/104248676/396716050-94a35a36-5685-421b-a2dd-bd8d84caca0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwMDEyNDIsIm5iZiI6MTczNTAwMDk0MiwicGF0aCI6Ii8xMDQyNDg2NzYvMzk2NzE2MDUwLTk0YTM1YTM2LTU2ODUtNDIxYi1hMmRkLWJkOGQ4NGNhY2EwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQwMDQyMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMGU5NzRkMDI5ZjM4ZmJkYjIyMTBkYjVmMGJkMGE2ZmU4ZGQ0ZjJiNGM0ZGIyMzcxYTIwNTQ3MDU4M2RkZmFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LCRV5wLt2IMbvmUGThaPpNiXMgAPmFkwBNHgCyfL4Yg)

I found it in: `%USERPROFILE%\AppData\Local\winutil\cttlogo.png` (note that it's a `.png` now).
It's likely only there if WinUtil has created the shortcut previously, so you may need to copy and save the icon from this post.
